crates/rethnet/tests/cli.rs

```diff
@@ -0,0 +1,179 @@
+use std::process::{Child, Command, Stdio};
+
+use assert_cmd::{
+    assert::Assert,
+    cargo::CommandCargoExt, // for process::Command::cargo_bin
+};
+use predicates::str::contains;
+
+use rethnet_eth::{
+    remote::{
+        client::Request as RpcRequest, jsonrpc, methods::MethodInvocation as EthMethodInvocation,
+        BlockSpec,
+    },
+    Address, Bytes, U256,
+};
+use rethnet_rpc_server::{HardhatMethodInvocation, MethodInvocation, TEST_ACCOUNTS};
+
+fn shutdown(child: &mut Child) -> Result<(), std::io::Error> {
```

Comment:
Is there a reason you're not using: https://doc.rust-lang.org/std/process/struct.Child.html#method.kill

Reply:
i think at first i was trying to simulate a ctrl+c(windows)/SIGINT(unix) rather than SIGTERM, thinking that that was more proper, and then along the way in my research i think i was led further astray by some crate that was offering a unix-only wrapper around the unix `kill` command, but eventually i settled on SIGTERM, but i hadn't realized `Child::kill` was a thing, but now i'm using it, as of [this commit](https://github.com/NomicFoundation/hardhat/commit/5ecc2971149d2938976e5c64f0cc4e279ed7105f)